iml-agent/src/rpm.rs

```diff
@@ -1,25 +1,51 @@
 use regex::Regex;
 
+use std::fmt;
 use std::process::Output;
 
 use crate::{agent_error::ImlAgentError, cmd::cmd_output};
 
-fn parse(output: Output) -> Result<bool, ImlAgentError> {
+#[derive(Debug, Eq, PartialEq, serde::Serialize, serde::Deserialize)]
+pub struct Version(String);
+
+impl fmt::Display for Version {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "{}", self.0)
+    }
+}
+
+async fn parse(output: Output) -> Result<Option<Version>, ImlAgentError> {
```

Comment:
Does this fn need to be `async`? I don't see any asynchronous work happening in it.

Reply:
Done